.readthedocs.yml

```diff
@@ -17,9 +17,6 @@ build:
   jobs:
     # See https://github.com/readthedocs/readthedocs.org/issues/4912
     pre_create_environment:
-      # Select Python version (keep in sync with other versions):
```

Comment:
Please, don't remove these lines. Because soon it will be updated and RTD will start to fail.